src/main/java/com/example/petbuddybackend/controller/AnimalController.java

```diff
@@ -0,0 +1,44 @@
+package com.example.petbuddybackend.controller;
+
+import com.example.petbuddybackend.service.animal.AnimalService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.security.SecurityRequirements;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/animal")
+public class AnimalController {
+
+    private final AnimalService animalService;
+
+    @SecurityRequirements
+    @Operation(summary = "Get animal attributes for specified animal type")
+    @GetMapping("attributes/{animalType}")
```

Comment:
```suggestion
    @GetMapping("/{animalType}/attributes")
```

Dałbym odwrotnie bo ma sens że w `/animal/{type}` podajesz `zwierze/typ/atrybut_dla_typu`

Reply:
Racja